contrib/osx/keybindings.el

```diff
@@ -30,4 +30,6 @@
                       (interactive)
                       (call-interactively (key-binding "\C-x\C-s"))))
     (global-set-key (kbd "s-Z") 'undo-tree-redo)
-    (global-set-key (kbd "C-s-f") 'spacemacs/toggle-frame-fullscreen)))
+    (global-set-key (kbd "C-s-f") 'spacemacs/toggle-frame-fullscreen)
+    ;; Emacs sometimes registers C-s-f as this weird keycode
+    (global-set-key (kbd "<C-s-268632070>") 'spacemacs/toggle-frame-fullscreen)))
```

Comment:
I have it in my own config, it's fine if its not merged. But as it is, OS X layer will be broken for some people. If they press command-shift-f, they will get an error that `<C-s-268632070>` is not bound to anything. For me, and for some other people, C-s-f registers as `<C-s-268632070>`. Instead of reeling at the ugly looking number (which I don't particularly *like*, as well), try understanding the problem, please.